bin/ghe-restore

```diff
@@ -352,7 +352,7 @@ else
   ghe-restore-mysql "$GHE_HOSTNAME" 1>&3
   # Clear GitHub Connect settings stored in the restored database
   if ! $RESTORE_SETTINGS; then
-    echo "if [ -f /usr/local/share/enterprise/ghe-reset-gh-connect ]; then /usr/local/share/enterprise/ghe-reset-gh-connect -y; else ghe-reset-gh-connect -y; fi" |
+    echo "if [ -f /usr/local/share/enterprise/ghe-reset-gh-connect ]; then /usr/local/share/enterprise/ghe-reset-gh-connect -y; fi" |
```

Comment:
My BASh is a bit rusty, this should maintain an open ssh connection to `$GHE_HOSTNAME` , right?

Reply:
Right, it will send this line of code into the SSH session, and will wait until it completes